package/boot/uboot-envtools/files/mediatek_filogic

```diff
@@ -37,6 +37,9 @@ bananapi,bpi-r3)
 		;;
 	esac
 	;;
+xiaomi,redmi-router-ax6000)
+	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x10000" "0x20000"
+	;;
 esac
 
 config_load ubootenv
```

Comment:
To support the `fw_setsys` and `fw_printsys` commands, change this line:
`config_foreach ubootenv_add_app_config ubootenv`
to
`config_foreach ubootenv_add_app_config`